### ISSUE TITLE
Fullscreen Mode

### ISSUE THREAD

**Issue by fabiantheblind** [basiljs/basil.js#84]
as mentioned by @ffd8 and @trych in #83 This could be nice
```js
function fullscreen(){
  // Preview mode with editing turned off.
  // Mouse clicks and arrow keys will move to previous or next spread.
  // see http://yearbook.github.io/esdocs/#/InDesign/ScreenModeOptions
    app.activeWindow.screenMode = ScreenModeOptions.PRESENTATION_PREVIEW;
  // app.activeWindow.zoom(ZoomOptions.FIT_SPREAD);
  // see http://yearbook.github.io/esdocs/#/InDesign/ZoomOptions
    app.activeWindow.zoom(ZoomOptions.FIT_PAGE);
}
fullscreen();
```

**Comment by trych** [comment]
Should we actually make this something like `b.screenMode()`? Then you have more options than just fullscreen. You could toggle between

PRESENTATION_PREVIEW
PREVIEW_OFF
PREVIEW_TO_PAGE

and so on (see the [ScreenModeOptions](http://www.indesignjs.de/extendscriptAPI/indesign8/#ScreenModeOptions.html)). Or would this require to introduce a `b.PREVIEW_CONSTANT` for each? Then maybe this might clutter things.

**Comment by ffd8** [comment]
My previous mention of fullScreen() was just a joke! Referring to the new/changing functions in Processing..

At a recent workshop, people with the latest version of InDesign were really disappointed that they couldn't watch any live rendering take place.. It just waited on the blank page until all was done. Tried a few setting tweaks, but same thing.. So I think a presentation view function could be fancy for super rare cases.. But I'd put it low on the totem pole of feature requests.

**Comment by trych** [comment]
>My previous mention of fullScreen() was just a joke!
Yes, I know it was a joke, but I actually think that such a function might be useful. For my projects I needed it and for interactive projects (Space Invaders, anyone?) it would also be great.

>At a recent workshop, people with the latest version of InDesign were really disappointed that they couldn't watch any live rendering take place.

Is that specific to something that has changed in the latest InDesign version? Because text is by default not rendered while a script is run, no matter which ID version. I have in a recent project overcome this issue by hacking some sort of presentation or – as I named it – kiosk mode. I access `textFrame.parentStory.lines.lastItem().baseline` to force-render a text frame that has been updated before. Read this trick in some really old article by Dave Saunders once. Additionally I put a lot of delay commands in my script, so it would be nicer to watch.

I think such kiosk mode features would also be cool to have, but I think we should open a separate issue for that. The `fullscreen()` method should be a function for itself, I think.
